azure/iac/Redis.rego

```diff
@@ -0,0 +1,95 @@
+package rule
+
+# https://docs.microsoft.com/en-us/azure/templates/Microsoft.Cache/redis
+
+# PR-AZR-0131-ARM
+
+default serverRole = null
+
+azure_attribute_absence ["serverRole"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.cache/redis"
+    not resource.properties.enableNonSslPort
+}
+
+azure_issue ["serverRole"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.cache/redis"
+    resource.properties.enableNonSslPort != false
+}
+
+serverRole {
+    azure_attribute_absence["serverRole"]
+}
+
+
+serverRole {
+    lower(input.resources[_].type) == "microsoft.cache/redis"
+    not azure_issue["serverRole"]
+}
+
+
+serverRole = false {
+    azure_issue["serverRole"]
+}
+
+serverRole_err = "Azure Redis Cache enableNonSslPort is not false currently on the resource" {
+    azure_issue["serverRole"]
+}
```

Comment:
add a else message when property does not exist. this can be a technical message.

Reply:
@rezoan 
If the property does not exist, we have a true result; so we don't need an error massage for that
but it if the property exists and the value of that was true, we have a false result, so we have an error massage

Reply:
got it.